stylesheet.css

```
@@ -0,0 +1,228 @@
+* {
+    box-sizing: border-box;
+    font-size: 62,5%;
+    margin: 0
+}
+
+*::after,
+*::before {
+    box-sizing: inherit;
+    padding: 0;
+}
+
+body {
+    background-color: #F9DB61FF;
+    font-family:Arial, Helvetica, sans-serif;
+}
+
+
+.header {
+    background-image: url(images/banner-bg.jpg);
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+    height: 100vh;
+    width: 100%;
+}
+    .header__main {
+        width: 100%;
+        height: 80%;
+        position: relative;
+    }
+    .header__main--box {
+        position: absolute;
+        top: 50%;
+        left: 50%;
+        transform: translate(-50%,-50%); /*centrowanie tekstu nagłówka*/
+    }
+    .heading {
+        font-size: 4.8em;
+        text-transform: uppercase;
+        text-align: center;
+    }
+    .heading--left {
+        color: white;
+    }
+    .heading--right {
+        color: black;
+    }
+    .header__main--paragraph {
+        display: flex;
+        color: gray;
+        margin-bottom: 5rem;
+        text-align: center;
+        font-size: 1em;
+    }
+
+
+.nav {
+    background-color: rgba(0, 0, 0, 0.2);
+    display:flex;
+    justify-content:space-around;
+    align-items: center;
+    min-height: 10vh;
+}
+    .nav--logo {
+        display:inline-block
+    }
+    .nav--links {

```

```suggestion
    .nav__links {

```
It is not a variant of `nav` but its valid part.